mail-api.js

```diff
@@ -96,6 +96,34 @@ module.exports = {
     });
   },
   
+
```

Comment:
We should include documentation comments so this can get added into the reference.